index.js

```diff
@@ -265,9 +266,22 @@ function argsToTokens(args, options) {
 
     ArrayPrototypePush(tokens, { kind: 'positional', index, value: arg });
   }
+
   return tokens;
 }
 
+/**
+ * Check if the given option that includes a default value
+ * has been set by the input args.
+ *
+ * @param {array} options - option configs entry, from parseArgs({ options })
+ * @param {object} values - option values returned in `values` by parseArgs
+ */
+function isDefaultValueOptionUsed({ 0: longOption, 1: optionConfig }, values) {
+  return optionConfig.defaultValue !== undefined &&
+  !objectGetOwn(values, longOption);
```

Comment:
This is ok, but overkill. We create `values` with a null prototype, so we don't need to worry about prototype pollution for it. Could just test for `values[longOption] != null` or `values[longOption] !== undefined`.

Reply:
Done